### PR TITLE
Reorganise the board

### PR DESCRIPTION
**Pull request by direnakkoc**
The issue applied:

- https://github.com/InnerSourceCommons/innersourcecommons.org/issues/556

### CODE REVIEW

**Comment by direnakkoc** [comment]
Hi @andrea-kyurchiev,
Could you please review the PR?

**Review by andrea-kyurchiev — APPROVED** [review]

**Comment by CiaraFlanagan** [comment]
Hello there @MaineC @dicortazar @rrrutledge @tsadler1988 @spier @mcobby @gruetter @yuhattor @voborgus 

We're looking to finalise and update the Board & Governance page and I believe we need you to approve before it can be merged.

If it's possible for one of you to have a look at this, that would be great.

File: content/en/about/board/_index.md [hunk 1]
```diff
@@ -97,10 +109,27 @@ Jacob Green, the founder of Mosslabs.io, is an open source artist, strategist &
...
+
+
+
+
+
+
+
+
+
+
```
Comment (rrrutledge) [inline content/en/about/board/_index.md]:
```suggestion
```

Comment (rrrutledge) [reply content/en/about/board/_index.md]: Remove extra spaces.

**Review by rrrutledge — APPROVED** [review]
Thank you ‼️

File: content/en/about/board/_index.md (hunk 1 above)
Comment (rrrutledge) [reply content/en/about/board/_index.md]: Looks good 👍 ✅❗️

**Comment by spier** [comment]
Thank you for these fixes @direnakkoc and @andrea-kyurchiev!

**Pull request merged**